packages/components/src/components/hive-layout-config.tsx

```diff
@@ -0,0 +1,15 @@
+/**
+ * @internal Don't expose this to websites.
+ */
+export const __LANDING_WIDTHS_ID = 'hive-l-widths';
+
+export interface HiveLayoutConfigProps {
+  widths: 'landing-narrow' | 'docs-wide';
+}
+
+/**
+ * @see {@link HiveLayout} from `@theguild/components/server` for documentation.
+ */
+export function HiveLayoutConfig({ widths }: HiveLayoutConfigProps) {
+  return widths === 'landing-narrow' ? <div id={__LANDING_WIDTHS_ID} /> : null;
```

Comment:
I decoupled this from the light mode override, because that light-mode include-list is _very_ error-prone, and we'd ideally get rid of it in the future.